modules/shortcodes/crowdsignal.php

```diff
@@ -209,7 +235,9 @@ function crowdsignal_shortcode( $atts ) {
 
 			$item_id = esc_js( $item_id );
 
-			if ( $inline ) {
+			if ( Jetpack_AMP_Support::is_amp_request() ) {
```

Comment:
Just a note that we'd need to add in support for this function before merging to WP.com.

Reply:
This would be handled by D22154-code .